src/dnc/azext_dnc/generated/commands.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from azext_dnc.generated._client_factory import cf_controller
+    dnc_controller = CliCommandType(
+        operations_tmpl='azext_dnc.vendored_sdks.dnc.operations._controller_operations#ControllerOperations.{}',
+        client_factory=cf_controller)
+    with self.command_group('dnc controller', dnc_controller, client_factory=cf_controller) as g:
+        g.custom_command('create', 'dnc_controller_create')
+        g.custom_command('delete', 'dnc_controller_delete', confirmation=True)
+        g.custom_command('patch', 'dnc_controller_patch')
```

Comment:
You should use `update` instead of `patch`

Reply:
I hide the patch method because our RP does not support it